proxy/development/nginx.conf

```diff
@@ -0,0 +1,94 @@
+user                       nginx;
```

Comment:
This file and the other have tabs that make them less readable imo. Nitpicking though so feel free to ignore

Reply:
Actually, I find it more readable when aligned with tab

Reply:
You're obviously wrong ^^